src/Repository/DefaultUnleashRepository.php

```diff
@@ -83,9 +90,9 @@ public function findFeature(string $featureName): ?Feature
      * @throws ClientExceptionInterface
      * @throws JsonException
      *
-     * @return iterable<Feature>
+     * @return array<Feature>
```

Comment:
I see we're shifted from an iterable to an array in a few places in this PR but I can't see why we would need to. What's happening here?